### PR TITLE
Feature/blog post page

### PR DESCRIPTION
**Pull request by anja-webcode**

### CODE REVIEW

File: incode-blog/next-env.d.ts [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /// <reference types="next" />
 /// <reference types="next/types/global" />
+/// <reference types="next-images" />
```
Comment (adriancova) [inline incode-blog/next-env.d.ts:3]: will we need this?

**Review by adriancova — APPROVED** [review]
🐝

File: incode-blog/next-env.d.ts (hunk 1 above)
Comment (anja-webcode) [reply incode-blog/next-env.d.ts:3]: Yeah...
"Typescript doesn't know how interpret imported images. next-images package contains definitions for image modules, you need to add reference to next-images types (third line) into your next-env.d.ts file."
[source](https://github.com/twopluszero/next-images#readme)

File: incode-blog/api/api.ts [hunk 2]
```diff
@@ -19,3 +19,11 @@ export async function getPosts() {
       console.error(err)
     })
 }
+
+export async function getSinglePost(postSlug: string) {
+  return await api.posts
+    .read({ slug: postSlug }, { include: ['tags'] })
+    .catch(err => {
+      console.error(err)
```
Comment (alexisllamas) [inline incode-blog/api/api.ts:27]: I don't think we should catch the error, if something fails probably we need to show that

File: incode-blog/.eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  ],
+  "parser": "@typescript-eslint/parser",
+  "settings": {
+    "react": {
+      "pragma": "React",
+      "version": "detect"
+    }
+  },
+  "rules": {
+    "react/prop-types": "off",
```
Comment (jitt3) [inline incode-blog/.eslintrc.json:22]: is it needed to turn proptypes off? as this project is using ts, I am not sure if this rules should be added here

**Review by jitt3 — APPROVED** [review]

**Pull request merged**